### PR TITLE
Add new DockerComposeRun builder

### PR DESCRIPTION
**Pull request by gabrielboliveira**
Even though it's not in use for one of our internal commands, this may be useful for custom commands, per-project.

### CODE REVIEW

File: app/Builder/DockerComposeRun.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+class DockerComposeRun extends Builder
+{
+    use HasEnvironmentVariables;
+
+    /** @var string $user */
+    protected $user = '';
+
+    public function getProgramName() : string
+    {
+        return 'run';
```
Comment (dbpolito) [inline app/Builder/DockerComposeRun.php]: this is the very same code as `DockerComposeExec`,  the only difference is this programname, maybe worth creating and abstract `DockerComposeBase`? 

Not sure if worth it, just wondering

File: .fwd [hunk 2]
```diff
@@ -29,6 +29,8 @@ FWD_DOCKER_BIN=docker
 FWD_DOCKER_COMPOSE_BIN=docker-compose
 # Docker Compose Exec flags
 FWD_COMPOSE_EXEC_FLAGS=""
+# Docker Compose Run flags
+FWD_COMPOSE_RUN_FLAGS="--rm"
```
Comment (dbpolito) [inline .fwd]: i think it may be good to default it to `-it --init` too

Comment (dbpolito) [reply .fwd]: also to follow docker run, `--rm` is always injected, so maybe add it to builder directly and on the custom flags just `-it --init`

https://github.com/fireworkweb/fwd/blob/ab88db2cb43e28ccf37d63fa9f016d4f445c5d43/app/Builder/DockerRun.php#L20

File: app/Builder/DockerComposeRun.php (hunk 1 above)
Comment (gabrielboliveira) [reply app/Builder/DockerComposeRun.php]: Not only this but the env variable also

File: .fwd (hunk 2 above)
Comment (gabrielboliveira) [reply .fwd]: `docker-compose run` doesn't have `-it` and `--init` flags:

https://docs.docker.com/compose/reference/run/

**Pull request merged**